### PR TITLE
SPCTR-3487: Fix - Inline-Notice title is not in  full width with Firefox browser

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description

Inline-Notice block title is not getting full width in Firefox browser.
https://brainstormforce.atlassian.net/browse/SPCTR-3487

### Screenshots

https://d.pr/i/dENZ3R

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -215,6 +215,9 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 
 ## Changelog ##
 
+### x.x.x ###
+* Fix: Inline Notice - Resolved an issue wherein the title width is not full in Firefox browser.
```
Comment (bsf-zanev) [inline README.md:219]:
```suggestion
* Fix: Inline Notice - Resolved an issue wherein the title would not appear to be full-width in the Firefox browser.
```

File: src/blocks/inline-notice/style.scss [hunk 2]
```diff
@@ -32,7 +32,7 @@
 
 	.uagb-notice-title {
 		margin: 0;
-		width: -webkit-fill-available;
+		width: -moz-available;
```
Comment (bsf-zanev) [inline src/blocks/inline-notice/style.scss:35]: Let's add the browser based overrides.

```suggestion
		width: -moz-available;
		width: -webkit-fill-available;
		width: fill-available;
```

**Review by bsf-zanev — COMMENTED** [review]
Added a minor changelog improvement, and a suggestion for cross-browser compatibility.

File: src/blocks/inline-notice/style.scss (hunk 2 above)
Comment (rajmohanreddy774) [reply src/blocks/inline-notice/style.scss:35]: Added browser based overrides.

File: README.md (hunk 1 above)
Comment (rajmohanreddy774) [reply README.md]: updated.

**Review by AbhijitNage123 — APPROVED** [review]

**Pull request merged**